host_metadata.go

```diff
@@ -0,0 +1,100 @@
+package mackerel
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// https://mackerel.io/ja/api-docs/entry/metadata
+
+// HostMetaDataResp represents response for host metadata.
+type HostMetaDataResp struct {
+	HostMetaData HostMetaData
+	LastModified time.Time
+}
+
+// HostMetaData represents host metadata body.
+type HostMetaData map[string]interface{}
```

Comment:
Host metadata accepts ANY *JSON (object, array, string, number, boolean, null)* as value, so this interface is not enough.
ref: https://mackerel.io/api-docs/entry/metadata